browse/services/document/author_affil.py

```diff
@@ -0,0 +1,363 @@
+"""Parse Authors lines to extract author and affiliation data."""
```

Comment:
should these go in `arxiv.base`? Seems of general use for arxiv-submission-ui and arxiv-search

Reply:
The plan for this was explicitly not to reuse the code in search for this. This decision could be revisited.